demo/js/MIDI/Loader.js

```diff
@@ -0,0 +1,388 @@
+/* 
```

Comment:
3rd party code without reference? Either use a git submodule or contribute to the project to release a npm package